mapdata/WarcraftLegacies/AbilityData/Skin/1634225729.json

```diff
@@ -23,7 +23,7 @@
       "Level": 2,
       "Id": 828536161,
       "Type": 3,
-      "Value": "Harvests lumber from trees."
+      "Value": "Harvest Gold from gold mines"
```

Comment:
Missed this ability! (Both this tooltip and the one below).

Reply:
Those are unused, they are the ghoul and arcjimonde ghoul abilities

Reply:
Cool, then let's delete those files entirely rather than modifying them!

Reply:
They are vanilla, can't delete AFAIK!

Reply:
You can. As far as the data structure of the map is concerned, vanilla abilities and custom ones are the same other than the fact that vanilla ones have a vanilla ID.

To "delete" it in the editor, right-click on it and do pick the "Reset to default settings" button.

Reply:
I see, Il take care of it!